### PR TITLE
Add Free Software

### PR DESCRIPTION
**Pull request by johnjago**
**https://github.com/johnjago/awesome-free-software**

**This is a list of common programs (as opposed to the other two FOSS lists which focus more on development tools) that have a Free Software Foundation approved license. The purpose of this list is to give users the ability to quickly find the best free software alternatives for many categories of applications.**
# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!
- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. _(For example: `Add Node.js`)_
- The list I added
  - [x] **has been around for at least 20 days,**
  - [x] is a non-generated Markdown file in a GitHub repo,
  - [x] is not a duplicate,
  - [x] only has awesome items with descriptions _(Awesome lists are curations of the best, not of everything)_,
  - [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
  - [x] has a Table of Contents section named `Contents` as the first section,
  - [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) _(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)_,
  - [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
  - [x] has been checked for inconsistent formatting, spelling, and grammar _(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)_,
  - [x] has been added at the bottom of the appropriate category.

### CODE REVIEW

**Comment by JesseTG** [comment]
I feel like you should mention in each entry what the license is, exactly.

**Comment by johnjago** [comment]
@JesseTG I thought that might make the list cluttered, especially since each entry already has the operating systems next to it. Instead, I added a few sentences pointing to a website that describes most free software licenses. Almost every linked homepage says the license, so it would be easy for a reader to check the website (http://choosealicense.com/licenses/) to find out more.

**Comment by JesseTG** [comment]
> I thought that might make the list cluttered, especially since each entry already has the operating systems next to it.

This is fine in itself, but it's not relevant for denoting free-ness.  I can write proprietary software for Linux, or I can contribute to Notepad++ -- which is free software but relies heavily on Windows APIs so it's not really suitable for Linux.

> Instead, I added a few sentences pointing to a website that describes most free software licenses. Almost every linked homepage says the license, so it would be easy for a reader to check the website (http://choosealicense.com/licenses/) to find out more.

Cool, that's fine.  My rationale is that free-software people are *really* picky about their licenses; personally, I'd rather just slap a license on the list entry than potentially get some poor sap's hopes up until they click the link and see that the license isn't compatible with their project.

**Comment by ugurozturk** [comment]
I think the list need to be grow much more.

**Comment by sindresorhus** [comment]
Nice intro in your awesome list :)

Feedback:

- Use the [SVG version](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg) of the CC0 license badge.
- I would like to see more than just apps. Include some blog posts, tutorials, links to community resources (chat, forums, communities), videos of talks, book, learning material, etc.

File: readme.md [hunk 1]
```diff
@@ -457,6 +457,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [ChatOps](https://github.com/exAspArk/awesome-chatops) - Managing technical and business operations through a chat.
 - [Falsehood](https://github.com/kdeldycke/awesome-falsehood) - Falsehoods programmers believe in.
 - [Domain-Driven Design](https://github.com/heynickc/awesome-ddd) - Software development approach for complex needs by connecting the implementation to an evolving model.
+- [Free Software](https://github.com/johnjago/Awesome-Free-Software) - Common applications that meet the free software criteria set by the FSF.
```
Comment (sindresorhus) [inline readme.md]: Maybe something shorter like:

> Free as in freedom.

-

Would prefer if the repo name was lowercase, like all others.

**Comment by johnjago** [comment]
@JesseTG I decided that your suggestion was a good idea based on what you mentioned. The licenses have been added.

@sindresorhus The SVG version of the CC0 badge is now in place. Thank you for the ideas on how to make the list better, such as adding more than just computer programs. I will continue exploring and adding more content.

File: readme.md (hunk 1 above)
Comment (davisonio) [inline readme.md]: @johnjago can you change the link here to be lowercase and the shorter description?

**Comment by sindresorhus** [comment]
> I would like to see more than just apps. Include some blog posts, tutorials, links to community resources (chat, forums, communities), videos of talks, book, learning material, etc.

Still needed.

**Comment by sindresorhus** [comment]
ping :)

**Pull request merged**